package.json

```diff
@@ -19,25 +19,27 @@
-    "test:config:build": "echo 'export let mode = \"production\";'   > ./src/__tests__/setup/config.js",
+    "test:config:dev": "echo 'export let mode = \"development\";' > ./src/__tests__/setup/config.js",
+    "test:config:build": "echo 'export let mode = \"production\";' > ./src/__tests__/setup/config.js",
     "test": "yarn test:config:dev && uvu src/__tests__ -b -i setup",
     "test:all": "yarn test:config:dev && uvu src/__tests__ -i setup",
-    "test:build": "yarn test:config:build && yarn build && uvu src/__tests__ -i setup",
-    "test:watch": "watchlist src src/__tests__ -- yarn test:all",
+    "test:build": "yarn test:config:build && yarn build-release && uvu src/__tests__ -i setup",
     "test:coverage": "c8 --include=src yarn test:all",
     "test:report": "c8 report --reporter=text-lcov > coverage.lcov",
-    "build": "rm -f public/wasm/* && yarn build-wasm-bindgen && yarn build-app && yarn docs",
+    "test:watch": "watchlist src src/__tests__ -- yarn test:all",
     "build-app": "cross-env NODE_ENV=production webpack",
-    "build-wasm-bindgen": "yarn build-wasm && wasm-bindgen --no-typescript src/wasi-rust/target/wasm32-wasi/release/wasi-gitportal.wasm --browser --out-dir . && yarn wbg-to-wasi && mv *.wasm public/wasm/ && mv wasi-* src",
-    "build-wasm": "cd src/wasi-rust/ && cargo build --release --target=wasm32-wasi && mkdir -p ../../public/wasm",
+    "build-dev": "yarn build-wasm-dev && yarn wasm-bindgen-dev && yarn wbg-to-wasi",
+    "build-release": "yarn build-wasm-release && yarn wasm-bindgen-release && yarn wbg-to-wasi && yarn build-app",
+    "build-wasm-dev": "cd src/wasi-rust/ && cargo build --target=wasm32-wasi && cd ../..",
+    "build-wasm-release": "cd src/wasi-rust/ && cargo build --release --target=wasm32-wasi && cd ../..",
+    "cleanup": "cd public && rm -rf wasm && mkdir wasm && cd .. && mv *.wasm public/wasm/ && mv wasi-* src/ && yarn docs",
     "dev": "webpack-dev-server --watch-content-base --content-base public",
-    "dev-wasm": "cd src/wasi-rust/ && cargo build --target=wasm32-wasi && mkdir -p ../../public/wasm",
-    "dev-wasm-bindgen": "yarn dev-wasm && wasm-bindgen --no-typescript src/wasi-rust/target/wasm32-wasi/debug/wasi-gitportal.wasm --debug --keep-debug  --out-dir . && yarn wbg-to-wasi && mv *.wasm public/wasm/ && mv wasi-* src",
-    "watch-wasm-bindgen": "watchlist src/wasi-rust -i src/wasi-rust/target -- yarn dev-wasm-bindgen",
-    "wbg-to-wasi": "./scripts/wbg_to_wasi.js --path ./wasi-js/ wasi-gitportal_bg.js wasi-gitportal_bg_wasi.js"
+    "wasm-bindgen-dev": "wasm-bindgen --no-typescript src/wasi-rust/target/wasm32-wasi/debug/wasi-gitportal.wasm --debug --keep-debug --out-dir .",
+    "wasm-bindgen-release": "wasm-bindgen --no-typescript src/wasi-rust/target/wasm32-wasi/release/wasi-gitportal.wasm --browser --out-dir .",
+    "watch-wasm-bindgen": "watchlist src/wasi-rust -i src/wasi-rust/target -- yarn wasm-bindgen-dev",
```

Comment:
The command `watch-wasm-bindgen` now only regenerates the bindings whereas it used to compiles the Rust to wasm. It used to run `dev-wasm-bindgen` which ran `dev-wasm` (compile Rust to wasm) and then `wasm-bindgen` (generate bindings). Poor naming I know!

I like the way you've re-organised the naming of the script commands, so to fix this I think you can just change it to:
```
    "watch-build-dev": "watchlist src/wasi-rust -i src/wasi-rust/target -- yarn build-dev",
```
And change the README from `watch-wasm-bindgen` to `watch-build-dev`.

Reply:
Yup. Perfect!